config/default.yml

```diff
@@ -139,6 +139,16 @@ AllCops:
   # from the lock file.) If the Ruby version is still unresolved, RuboCop will
   # use the oldest officially supported Ruby version (currently Ruby 2.4).
   TargetRubyVersion: ~
+  # Determines if a notification for extension libraries should be shown when
+  # rubocop is run. Keys are the name of the extension, and values are an array
+  # of gems in the Gemfile that the extension is suggested for, if not already
+  # included.
+  NotifyForExtensions:
```

Comment:
I'm also wondering if `SuggestExtensions` is not better terminology.

Reply:
Works for me!